crates/biome_cli/src/commands/init.rs

```diff
@@ -12,12 +12,12 @@ pub(crate) fn init(mut session: CliSession) -> Result<(), CliDiagnostic> {
 
 "<Info><Emphasis>"Files created "</Emphasis></Info>{HorizontalLine::new(106)}"
 
-  "<Dim>"- "</Dim><Emphasis>"biome.json: "</Emphasis>"Your project configuration. Documentation: "<Hyperlink href="https://biomejs.dev/configuration">"https://biomejs.dev/configuration"</Hyperlink>"
+  "<Dim>"- "</Dim><Emphasis>"biome.json: "</Emphasis>"Your project configuration. Documentation: "<Hyperlink href="https://biomejs.dev/reference/configuration">"https://biomejs.dev/reference/configuration"</Hyperlink>"
```

Comment:
This link wasn't mentioned in the issue but obviously I tested everything and this one was broken too